### PR TITLE
Feature/pe cases card

### PR DESCRIPTION
**Pull request by claudilla**

### CODE REVIEW

File: src/pe-cases-card/config.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  "common-name": "Cases Card",
+  "compatibility": ["London", "Madrid", "New York"],
+  "contributors": ["Claudia Cruz"],
+  "keywords": ["Widget"],
+  "name": "pe-cases-card",
+  "publishing-goal": "Update set",
+  "solution-details": "A modern card widget that allows users to visualize the details of active cases.",
+  "type": "Widget",
+  "version": "1.0.0"
+}
```
Comment (stevengregory) [inline src/pe-cases-card/config.json]: Looks good. 👍

Comment (claudilla) [reply src/pe-cases-card/config.json]: Thanks :)

File: src/pe-cases-card/widget/pe-cases-card.u-server-script.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+(function() {
+	/* populate the 'data' object */
+	/* e.g., data.table = $sp.getValue('table'); */
+
```
Comment (stevengregory) [inline src/pe-cases-card/widget/pe-cases-card.u-server-script.js]: Please update the table option name and provide a fallback table; in the case a table isn't specified in the options.

File: src/pe-cases-card/widget/pe-cases-card.u-client-script.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
+function CaseCardsController() {
```
Comment (stevengregory) [inline src/pe-cases-card/widget/pe-cases-card.u-client-script.js]: Thanks for adding the ctrl named function. 👍

File: src/pe-cases-card/widget/pe-cases-card.u-option-schema.json [hunk 4]
```diff
@@ -0,0 +1 @@
+[{"name":"table","section":"other","default_value":"sn_customerservice_case","label":"table","type":"string"}]
```
Comment (stevengregory) [inline src/pe-cases-card/widget/pe-cases-card.u-option-schema.json]: Table option name needs to be updated to operate correctly.

File: src/pe-cases-card/widget/pe-cases-card.u-css.scss [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (stevengregory) [inline src/pe-cases-card/widget/pe-cases-card.u-css.scss]: Please consider removing redundant space in the future.

File: package-lock.json [hunk 6]
```diff
@@ -4,6 +4,1062 @@
   "lockfileVersion": 1,
   "requires": true,
   "dependencies": {
+    "acorn": {
```
Comment (stevengregory) [inline package-lock.json]: Package updates don't need to be captured in this PR.

**Pull request merged**